src/components/atoms/ReviewCardCarousel/ReviewCard.module.scss

```diff
@@ -0,0 +1,35 @@
+.card-container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  justify-content: center;
+  width: 100%;
+}
+
+.card-img {
+  border: 5px solid #c33; /* Shortened the hex color */
+  border-radius: 100px; /* Removed the percentage unit for border-radius */
```

Comment:
You can use our border radius variable here. Visit our foundation folder. lease refer to our foundation folder. We primarily employ CSS variables here and define them.